requirements-unlocked.txt

```diff
@@ -1 +1,7 @@
+beautifulsoup4
```

Comment:
@joeflack4 why is this needed?

Reply:
Very observant! I was wondering if you would notice this.

There is a utility which we haven't used in years called the code scraper. It is described in the "Additional tools" section of the readme. It allows us to see which MIMs had changes by a given month, and it pulls this information from the website.